config/seata-config-core/src/main/java/io/seata/config/ConfigFuture.java

```diff
@@ -99,13 +99,10 @@ public Object get(long timeout, TimeUnit unit) {
         } catch (ExecutionException e) {
             throw new ShouldNeverHappenException("Should not get results in a multi-threaded environment", e);
         } catch (TimeoutException e) {
-            LOGGER.error(
-                    "config operation timeout,cost:" + (System.currentTimeMillis() - start) + " ms,op:" + operation
-                            .name()
-                            + ",dataId:" + dataId);
+            LOGGER.error("config operation timeout,cost:{} ms,op:{},dataId:{}", System.currentTimeMillis() - start, operation.name(), dataId);
             return getFailResult();
         } catch (InterruptedException exx) {
-            LOGGER.error("config operate interrupted,error:" + exx.getMessage());
+            LOGGER.error("config operate interrupted,error:{}", exx.getMessage());
```

Comment:
Please call the method with three parameters.LOGGER.error("xxxx:{}",exx.getMessage(),exx)

Reply:
Received, I will fix this.